common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -131,7 +131,7 @@ def verify_demand_hints(self, first_hint, second_hint, expected_events):
         problem_page = ProblemPage(self.browser)
 
         # The single visible Hint button should be enabled.
-        self.assertEqual([None], problem_page.get_hint_button_disabled_attr())
+        self.assertEqual([None, None], problem_page.get_hint_button_disabled_attr())
```

Comment:
Change the comment right above, and can you test that the hint notification is not initially visible (though now the HTML exists)?